espanol/batman/Batman.md

```diff
@@ -0,0 +1 @@
+No puedes ser Batman cuando yo soy el verdadero Batman, sigue y se un [Robin] (robin/robin.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.